### PR TITLE
Refactor/components

### PR DESCRIPTION
**Pull request by Paalar**
The point of this PR is to split up the code to make it hopefully more readable, easier to get into, and write more deletable code.
- Several of the atoms were used in only 1 specific component, which has now been moved into a composite folder
- Large components has now been split up, which makes it easier to get into.
- Other minor fixes, like dependency verisons and renaming

### CODE REVIEW

File: src/components/ApplicationWindow/Window.tsx [hunk 1]
```diff
@@ -1,49 +1,46 @@
...
 `;
 
 type WindowProps = {
   className?: string;
   name: string;
-  onClick: Function;
   onClose?: () => void;
-  // eslint-disable-next-line no-undef
-  children: JSX.Element[] | JSX.Element;
+  children: JSX.Element | JSX.Element[];
```
Comment (kharann) [inline src/components/ApplicationWindow/Window.tsx:27]: Dont think this is necesarry to define children. By default the FC type (which you have at window) should include children by default. It also has its own type.

File: src/components/Basket/BasketStatus.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                src={`${process.env.PUBLIC_URL}/${cartImageUri}.png`}
+                alt="Empty folder"
+            />
+            Basket
+            <TotalDisplay total={total}>({total} NOK)</TotalDisplay>
+        </h3>
+    );
+};
+
+const TotalDisplay = styled.span`
```
Comment (kharann) [inline src/components/Basket/BasketStatus.tsx:24]: Might be an idea to define styled props in here?

Like `const TotalDisplay = styled.span<BasketStatusProps>`

**Review by kharann — APPROVED** [review]
LGTM otherwise

File: src/components/ApplicationWindow/Window.tsx (hunk 1 above)
Comment (Paalar) [reply src/components/ApplicationWindow/Window.tsx]: Fixed by removing it.

File: src/components/Basket/BasketStatus.tsx (hunk 2 above)
Comment (Paalar) [reply src/components/Basket/BasketStatus.tsx]: Could not add BasketStatus as it does not receive cart, but only total.

**Pull request merged**